src/logic/addressBook/store/__tests__/addressBookReducer.test.ts

```diff
@@ -7,12 +7,12 @@ describe('Test AddressBook BatchLoadEntries Reducer', () => {
     it('returns an addressbook array', () => {
       const addressBookEntries = [
         {
-          address: '0x4462527986c3fD47f498eF25B4D01e6AAD7aBcb2',
+          address: '0x5fb582FD320ab1CBf055F65ED74D01b9DdB90A00',
```

Comment:
All addresses were updated in this test file as they were invalid addresses.

Reply:
Should only checksummed addresses be imported? If so, should we checksum them before saving an entry so that when they are exported, they can be imported again?

Reply:
Good idea!